go.mod

```diff
@@ -194,7 +194,7 @@ require (
 	k8s.io/klog/v2 v2.100.1
 	k8s.io/kubectl v0.27.4
 	sigs.k8s.io/controller-runtime v0.15.0
-	sigs.k8s.io/controller-tools v0.12.0
+	sigs.k8s.io/controller-tools v0.12.1
```

Comment:
This might be breaking, although likely it's fine:

> 🐛 Change the type of markers.Collector.byPackage's key from string to *loader.Package by @​k8s-infra-cherrypick-robot in kubernetes-sigs/controller-tools#834